data/transactions/logic/assembler_test.go

```diff
@@ -2576,3 +2576,22 @@ done:
  concat
 `, LogicVersion, Expect{5, "concat arg 1 wanted type []byte..."})
 }
+
+func TestMergeProtos(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	t.Parallel()
+	oneIntArgOneIntRet := OpSpec{Proto: proto("i:i")}
+	oneBytesArgOneBytesRet := OpSpec{Proto: proto("b:b")}
+	twoAnyArgsOneAnyRet := OpSpec{Proto: proto("aa:a")}
+	oneAnyArgsTwoAnyRets := OpSpec{Proto: proto("a:aa")}
```

Comment:
minor nit: might be easier to read the actual tests with var names like `i_i`, `b_b`, `aa_a`, `a_aa`

Reply:
Good point, done